<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalSetting">
      <attribute name="icon">
       <iconset resource="MainWindow.qrc">
        <normaloff>:/resource/home.png</normaloff>:/resource/home.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="autoStartGroup">
         <property name="title">
          <string>Auto Start</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="enableAutoStart">
            <property name="text">
             <string>Enable Auto Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disableAutoStart">
            <property name="text">
             <string>Disable Auto Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Auto start script:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PathEdit" name="autoStartScript" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="notices">
         <property name="title">
          <string>Notices</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Script start running:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="noticeScriptStart">
            <property name="text">
             <string>enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Script finish running:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="noticeScriptFinish">
            <property name="text">
             <string>enable</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Script runs failed:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="noticeScriptFailed">
            <property name="text">
             <string>enable</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="access">
         <property name="title">
          <string>Access</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Require password:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="enablePassword">
            <property name="text">
             <string>enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="password">
            <property name="inputMask">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scriptSetting">
      <attribute name="icon">
       <iconset resource="MainWindow.qrc">
        <normaloff>:/resource/file-code.png</normaloff>:/resource/file-code.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Script</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTreeView" name="scriptSettingView">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="settingButtons">
         <item>
          <widget class="QPushButton" name="newSetting">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteSetting">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="browserSetting">
      <attribute name="icon">
       <iconset resource="MainWindow.qrc">
        <normaloff>:/resource/browser.png</normaloff>:/resource/browser.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Browser</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="defaultValueGroup">
         <property name="title">
          <string>Default values</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Default Page Url:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="browserDefaultUrl"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="schedule">
      <attribute name="icon">
       <iconset resource="MainWindow.qrc">
        <normaloff>:/resource/schedule.png</normaloff>:/resource/schedule.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Schedule</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableView" name="scheduleView">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="newTask">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteTask">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonArea">
     <item>
      <widget class="QPushButton" name="submit">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PathEdit</class>
   <extends>QWidget</extends>
   <header>PathEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>deleteSetting</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>deleteSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newSetting</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>newSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submit</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAutoStart</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>enableAutoStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disableAutoStart</sender>
   <signal>clicked()</signal>
   <receiver>SettingPage</receiver>
   <slot>disableAutoStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset()</slot>
  <slot>submit()</slot>
  <slot>newSetting()</slot>
  <slot>deleteSetting()</slot>
  <slot>enableAutoStart()</slot>
  <slot>disableAutoStart()</slot>
 </slots>
</ui>
